source/hello.js

```diff
@@ -10,10 +14,14 @@ export default React => stampit(React, {
   propTypes: {
     actions: PropTypes.shape({
       setWord: func.isRequired,
-      setMode: func.isRequired,
```

Comment:
Trailing commas are automatically removed by babel. Please change the trailing comma rule. =)